### PR TITLE
Fixes bug and add method to delete recur and startTime

### LINKED ISSUE #111: add notRecur command

### LINKED ISSUE #112: implement a way to remove startTime

### PR DESCRIPTION
**Pull request by AiwassPrime**

### CODE REVIEW

**Review by yhtMinceraft1010X — COMMENTED** [review]
LGTM

**Comment by AiwassPrime** [comment]
Now can delete recur and startTime by using editCommand (just like tags).
Example: `edit 1 a/`
This will delete the startTime of task 1.

File: src/main/java/seedu/module/logic/parser/EditCommandParser.java [hunk 1]
```diff
@@ -95,4 +95,28 @@ public EditCommand parse(String args) throws ParseException {
...
+        assert recur != null;
+
+        if (recur.isNull()) {
+            return Optional.empty();
+        }
+
+        return Optional.of(recur.getField().equals("")
+                ? new OptionalField<>(null)
+                : new OptionalField<>(ParserUtil.parseRecurrence(recur.getField())));
+    }
```
Comment (umergta) [inline src/main/java/seedu/module/logic/parser/EditCommandParser.java:120]: looks good!

File: src/main/java/seedu/module/model/task/Task.java [hunk 2]
```diff
@@ -74,6 +74,17 @@ public static Task setTags(Task task, Set<Tag> tags) {
                 task.doneStatus, task.recurrence, tags);
     }
 
+    /**
+     * Factory method designed for tag and deleteTag command, to get rid of repeat code.
+     *
```
Comment (umergta) [inline src/main/java/seedu/module/model/task/Task.java]: think need to change this java doc!

**Review by umergta — COMMENTED** [review]
LGTM. just a few knits to fix

**Review by QY-H00 — COMMENTED** [review]
LGTM.

**Pull request merged**